### DISCUSSION TITLE
[灌水]大型纪录片《等星见雅》每周日持续为您播出！

### DISCUSSION THREAD

**Discussion by guiyu0424** [share121/inter-knot#4894] [绝区零]
安比！我的安比！没有你我可怎么活啊！
![Uploading Screenshot_2024-09-15-14-18-20-692_com.miHoYo.Nap-edit.jpg…]()

**Comment by guiyu0424** [comment]
![Screenshot_2024-09-15-14-18-20-692_com miHoYo Nap-edit](https://github.com/user-attachments/assets/18ec7033-0582-421c-8272-3f37cb47f387)

**Comment by Minapocalis** [comment]
你这是等星见雅？啪啪，这巴掌是为她打的
![Uploading 1724803950788.jpg…]()

**Reply by guiyu0424** [reply]
不，你不懂，她们都是我的翅膀！

**Reply by Minapocalis** [reply]
此时此刻，正如那句话说的那样，你下贱你那是喜欢吗？你那是馋她们的身子，
![1726224071696](https://github.com/user-attachments/assets/7a0dc2cf-8102-4aab-827a-51ee7bc980e7)
